thread/src/components/ResetPasswordForm.js

```diff
@@ -33,39 +45,54 @@ class ForgotPasswordForm extends Component {
         }
 
         return (
-            <Button full style={{ marginTop: 25 }} onPress={this.onSubmitPress.bind(this)}>
+            <Button rounded style={styles.buttonStyle} onPress={this.onSubmitPress.bind(this)}>
                 <Text>Reset password</Text>
             </Button>
         );
     }
 
     render() {
         return(
-            <Container style={{ backgroundColor: '#fff' }}>
+            <Container style={{ backgroundColor:'#8bc34a' }}>
                 <Content>
-                    <Form>
-                        <Item stackedLabel>
-                            <Label>Email address</Label>
+                    <Form style={{padding:20, marginLeft: -20}}>
+                        <Item style={{marginTop:40}} stackedLabel>
+                            <TextStyle>Email address</TextStyle>
                             <Input
+                                style={{fontSize:13, color:'white'}}
                                 onChangeText={this.onFirstEmailChanged.bind(this)}
                                 value={this.props.first_email}
                             />
                         </Item>
-                        <Item stackedLabel>
-                            <Label>Re-enter email address</Label>
+                        <Item style={{marginTop:40}}  stackedLabel>
+                            <TextStyle>Re-enter email address </TextStyle>
                             <Input
+                                 style={{ fontSize:13, color:'white'}}
                                 onChangeText={this.onSecondEmailChanged.bind(this)}
                                 value={this.props.second_email}
                             />
                         </Item>
+                            {this.renderSubmitButton()}
                     </Form>
-                    {this.renderSubmitButton()}
+
                 </Content>
             </Container>
         );
     }
 }
 
+let styles = StyleSheet.create({
+
+     buttonStyle: {
+        marginTop: 25,
+        alignItems: 'center',
+        justifyContent: 'center',
+        marginLeft: 20,
+        width: 340
```

Comment:
you are still using static widths

<img width="335" alt="screen shot 2018-06-19 at 8 50 00 pm" src="https://user-images.githubusercontent.com/22198040/41620633-3d5f8cb2-7402-11e8-89a4-f3fed7caa3ea.png">

Reply:
use percentages for width, or add a right margin